### PR TITLE
[DOC] editorial updates to DB Client guide

### PR DESCRIPTION
**Pull request by ljamen**
For issue #4497

### CODE REVIEW

**Review by m0mus — APPROVED** [review]

**Comment by m0mus** [comment]
@ljamen It doesn't build. Please fix.

File: docs/se/guides/dbclient.adoc [hunk 1]
```diff
@@ -116,49 +115,49 @@ docker build -f Dockerfile.h2 . -t h2db
...
 java -jar h2-\{latest-version}.jar -webAllowOthers -tcpAllowOthers
 ----
 
-==== Run H2
-
-This output should be printed:
 
 [source,bash]
-.Terminal output
+.Print the terminal output
```
Comment (tomas-langer) [inline docs/se/guides/dbclient.adoc:138]: This is the expected output of the command entered above. The verb "Print" seems to change the meaning (e.g. I would expect to be asked to print it).

**Comment by ljamen** [comment]
@m0mus I had the wrong path for an image. It should be fixed in the next build.

**Review by tomas-langer — APPROVED** [review]

**Pull request merged**